shared/actions/favorite.js

```diff
@@ -164,6 +164,11 @@ const jsonToFolders = (json: Object, myKID: any) => {
 
 export function favoriteList (): (dispatch: Dispatch, getState: () => Object) => void {
   return (dispatch, getState) => {
+    // Ignore any messages we receive while reloading
```

Comment:
should this happen at a higher level, like in the middleware? seems like we only want the dev actions / reducer to work while we're reloading?